pkgs/applications/editors/eclipse/default.nix

```diff
@@ -10,13 +10,16 @@ assert stdenv ? glibc;
 
 # https://download.eclipse.org/eclipse/downloads/ is the main place to
 # find the downloads needed for new versions
+#
+# to test:
+# $ for e in cpp modeling platform sdk java committers; do nix build -f default.nix pkgs.eclipses.${e} -o eclipses.${e}; done
```

Comment:
I think a `eclipse-` is missing here:
```suggestion
# $ for e in cpp modeling platform sdk java committers; do nix build -f . pkgs.eclipses.eclipse-${e} -o eclipses.${e}; done
(also default.nix could be simplified to .)
```

Reply:
Um ... it was right in my `.bash_history`?  Thanks!  :facepalm: